app/jobs/ingest_job.rb

```diff
@@ -48,12 +48,16 @@ def perform(model_params:, background_job_result:, signed_ids: {}, globus_ids: {
     # Create workflow destroys existing steps if called again, so need to check if already created.
     Workflow.create_unless_exists(druid, 'registrationWF', version: 1, priority: priority)
 
-    StageBlobs.stage(signed_ids, druid) do
-      Workflow.create_unless_exists(druid, 'accessionWF', version: 1, priority: priority) if start_workflow
-    end
-
-    StageGlobus.stage(globus_ids, druid) do
-      Workflow.create_unless_exists(druid, 'accessionWF', version: 1, priority: priority) if start_workflow
+    if signed_ids.any?
+      StageBlobs.stage(signed_ids, druid) do
+        Workflow.create_unless_exists(druid, 'accessionWF', version: 1, priority: priority) if start_workflow
+      end
+    elsif globus_ids.any?
+      StageGlobus.stage(globus_ids, druid) do
+        Workflow.create_unless_exists(druid, 'accessionWF', version: 1, priority: priority) if start_workflow
+      end
+    elsif start_workflow
+      Workflow.create_unless_exists(druid, 'accessionWF', version: 1, priority: priority)
```

Comment:
I wasn't sure if this was needed. Do we ever want to kick off accessioning when there are no files to stage?